packer/plugin/server.go

```diff
@@ -104,15 +91,28 @@ func Server() (*packrpc.Server, error) {
 	return packrpc.NewServer(conn)
 }
 
-func serverListener(minPort, maxPort int64) (net.Listener, error) {
+func serverListener() (net.Listener, error) {
 	if runtime.GOOS == "windows" {
-		return serverListener_tcp(minPort, maxPort)
+		return serverListener_tcp()
 	}
 
 	return serverListener_unix()
 }
 
-func serverListener_tcp(minPort, maxPort int64) (net.Listener, error) {
+func serverListener_tcp() (net.Listener, error) {
+	minPort, err := strconv.ParseInt(os.Getenv("PACKER_PLUGIN_MIN_PORT"), 10, 32)
+	if err != nil {
+		return nil, err
+	}
+
+	maxPort, err := strconv.ParseInt(os.Getenv("PACKER_PLUGIN_MAX_PORT"), 10, 32)
+	if err != nil {
+		return nil, err
+	}
+
+	log.Printf("Plugin minimum port: %d\n", minPort)
+	log.Printf("Plugin maximum port: %d\n", maxPort)
```

Comment:
Would it be a good idea to regroup these two lines into one ? Like I did in #8423 ?

Reply:
Yeah, I see no reason not to.